scripts/lib/install

```diff
@@ -13,49 +22,60 @@ EOF
 
 mkdir -p /etc/zulip
-echo -e "[machine]\npuppet_classes = zulip::voyager\ndeploy_type = voyager" > /etc/zulip/zulip.conf
+
+echo -e "[machine]\npuppet_classes = zulip::$VOYAGER_CLASS\ndeploy_type = voyager" > /etc/zulip/zulip.conf
 /root/zulip/scripts/zulip-puppet-apply -f
 
 # These server restarting bits should be moveable into puppet-land, ideally
 apt-get -y upgrade
-if [ -e "/etc/init.d/nginx" ]; then
-    # Check nginx was configured properly now that we've installed it.
-    # Most common failure mode is certs not having been installed.
-    nginx -t
-    service nginx restart
+# Don't start services when deploying in docker aka dockervoyager
+if [ "$DEPLOYMENT_TYPE" != "dockervoyager" ]; then
+    if [ -e "/etc/init.d/nginx" ]; then
+        # Check nginx was configured properly now that we've installed it.
+        # Most common failure mode is certs not having been installed.
+        nginx -t
+        service nginx restart
+    fi
+    /root/zulip/scripts/setup/generate_secrets.py
+    if ! rabbitmqctl status >/dev/null; then
+        set +x
+        echo; echo "RabbitMQ seems to not have started properly after the installation process."
+        echo "Often, this can be caused by misconfigured /etc/hosts in virtualized environments"
+        echo "See https://github.com/zulip/zulip/issues/53#issuecomment-143805121"
+        echo "for more information"
+        echo
+        set -x
+        exit 1
+    fi
 fi
-
-/root/zulip/scripts/setup/generate_secrets.py
 cp -a /root/zulip/zproject/local_settings_template.py /etc/zulip/settings.py
 ln -nsf /etc/zulip/settings.py /root/zulip/zproject/local_settings.py
+if [ "$DEPLOYMENT_TYPE" != "dockervoyager" ]; then
+    /root/zulip/scripts/setup/configure-rabbitmq
 
-if ! rabbitmqctl status >/dev/null; then
-    set +x
-    echo; echo "RabbitMQ seems to not have started properly after the installation process."
-    echo "Often, this can be caused by misconfigured /etc/hosts in virtualized environments"
-    echo "See https://github.com/zulip/zulip/issues/53#issuecomment-143805121"
-    echo "for more information"
-    echo
-    set -x
-    exit 1
+    /root/zulip/scripts/setup/postgres-init-db
+else
+    echo; echo "Skipping service starts, chosen Deployment Type is \'$DEPLOYMENT_TYPE\'."
 fi
 
-/root/zulip/scripts/setup/configure-rabbitmq
-
-/root/zulip/scripts/setup/postgres-init-db
-
 deploy_path=$(/root/zulip/zulip_tools.py make_deploy_path)
 mv /root/zulip "$deploy_path"
 ln -nsf /home/zulip/deployments/next /root/zulip
 ln -nsf "$deploy_path" /home/zulip/deployments/next
 ln -nsf "$deploy_path" /home/zulip/deployments/current
 ln -nsf /etc/zulip/settings.py "$deploy_path"/zproject/local_settings.py
+if [ "$DEPLOYMENT_TYPE" == "dockervoyager" ]; then
+    # We generate the static files later in the entrypoint.sh we don't have the
+    # static files right now, so just create the folder to fix the error with "cp ..."
+    mkdir -p "$deploy_path"/prod-static/serve
+fi
```

Comment:
Is this still needed with the new model of using the static assets from a release tarball?